src/Modules/Admin/Commands.Admin/RotatePowerBIEncryptionKey.cs

```diff
@@ -0,0 +1,78 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.PowerBI.Api.V2.Models;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Admin
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    public class RotatePowerBIEncryptionKey : PowerBIClientCmdlet
+    {
+        public const string CmdletName = "PowerBIEncryptionKey";
+        public const string CmdletVerb = "Rotate";
+
+        public RotatePowerBIEncryptionKey() : base() { }
+
+        public RotatePowerBIEncryptionKey(IPowerBIClientCmdletInitFactory init) : base(init) { }
+
+        #region Parameters
+
+        [Parameter(Mandatory = true)]
+        public string Name { get; set; }
```

Comment:
You may want to check null or empty on the strings

Reply:
Same for other cmdlets

Reply:
Powershell cmdlet by default checks for empty string and returns errors